go/identify3/adapter.go

```diff
@@ -264,6 +264,7 @@ func (i *UIAdapter) rowPartial(mctx libkb.MetaContext, proof keybase1.RemoteProo
 			}
 		}
 		row.ProofURL = i.makeSigchainViewURL(mctx, proof.SigID)
+		iconKey = serviceType.GetLogoKey()
```

Comment:
this line is crashing my service locally when `serviceType` is nil. it's a test user using a service type that no longer exists in the configs because my local setup is a little janky. but it seems like this could also reappear in prod if we ever remove a config (which we definitely might could do).

Reply:
Does https://github.com/keybase/client/pull/17169 fix that?